tendrl/commons/objects/cluster/atoms/delete_monitoring_details/__init__.py

```diff
@@ -36,9 +37,15 @@ def run(self):
             payload=payload
         ).save()
 
-        # Wait for 2 mins for the job to complete
+        # Find number jobs in a queue
+        no_of_jobs = 0
+        queue = etcd_utils.read("/queue")
+        for _ in queue.leaves:
```

Comment:
on `queue.leaves` there should be a way to get length using iterators I feel

Reply:
you can use `len(queue._children)` as that `_children` returns a list

Reply:
_children is a private variable, is it correct to use this in our code?